test/visual/test.js

```diff
@@ -28,6 +28,15 @@ gemini.suite('vaadin-select', function(rootSuite) {
         .setCaptureElements('#select')
         .capture('select');
     });
+
+    ['ltr', 'rtl'].forEach(dir => {
+      gemini.suite(`${dir}-align-tests-${theme}`, function(suite) {
+        suite
+          .setUrl(`align-themes.html?theme=${theme}&dir=${dir}`)
```

Comment:
One more note: let's only add these tests for Lumo. We don't have `align` themes in Material.

Reply:
Updated, now tested only for `lumo`